src/plugins/vis_builder/public/application/components/data_tab/field_bucket.tsx

```diff
@@ -60,14 +60,14 @@ export function FieldBucket({ bucket, field }: FieldBucketProps) {
             <EuiFlexItem grow={1} className="eui-textTruncate">
               <EuiText
                 title={`${displayValue}: ${count} (${percent.toFixed(1)}%)`}
-                size="xs"
+                size="s"
                 className="eui-textTruncate"
               >
                 {displayValue}
               </EuiText>
             </EuiFlexItem>
             <EuiFlexItem grow={false} className="eui-textTruncate">
-              <EuiText color="secondary" size="xs" className="eui-textTruncate">
+              <EuiText color="secondary" size="s" className="eui-textTruncate">
```

Comment:
I think we likely want to keep these if they are in a data popover

Reply:
will do